### PR TITLE
Update spur meta

### PR DESCRIPTION
**Pull request by krishnasism**

### CODE REVIEW

File: modules/sfp_spur.py [hunk 1]
```diff
@@ -19,12 +19,25 @@ class sfp_spur(SpiderFootPlugin):
...
+        'summary': "Obtain information about any malicious activities involving IP addresses found",
+        'flags': [ "apikey" ],
+        'useCases': [ "Investigate", "Passive" ],
+        'categories': [ "Reputation Systems" ],
+        'dataSource': {
+            'website': "https://spur.us/",
+            'model': "COMMERCIAL ONLY",
+            'references': [
+                "https://spur.us/api"
+            ],
```
Comment (smicallef) [inline modules/sfp_spur.py:32]: Missing instructions?

Comment (krishnasism) [reply modules/sfp_spur.py:32]: Couldn't go past the paywall, would require your input

Comment (smicallef) [reply modules/sfp_spur.py:32]: ok -

1. Pay for a subscription
2. Under the Context API menu on the left, click Tokens
3. Your API key is presented to you

Comment (krishnasism) [reply modules/sfp_spur.py:32]: Thanks, updated

**Pull request merged**